src/tx-builder/services/MerkleDistributor.ts

```diff
@@ -152,6 +154,40 @@ export default class MerkleDistributor extends BaseService<MerkleDistributorV1>
     return this.contract.hasPendingRoot();
   }
 
+  public async getUserBalancesPerEpoch(): Promise<UserRewardsBalancesPerEpoch> {
+    const activeRootData: ActiveRootDataAndHistory = await this.getActiveRootData();
+    return activeRootData.userBalancesPerEpoch;
+  }
```

Comment:
Why do we need this function? It seems to be used as a passthrough, and tbh this PR doesn't need to add tests for `getActiveRootData` (already have extensive tests in `governance-private` plus it has been in production for a while now).

Reply:
see above - removed